malaria-ios/ViewControllers/UserProfileViewController.swift

```diff
@@ -230,6 +302,26 @@ extension UserProfileViewController: SaveRemainingPillsProtocol {
   }
 }
 
+// MARK: Text Field Delegate
+
+extension UserProfileViewController: UITextFieldDelegate {
+  
+  func textFieldShouldReturn(textField: UITextField) -> Bool {
+    self.view.endEditing(true)
+    return false
+  }
+  
+  func textFieldDidBeginEditing(textField: UITextField) {
+    let newOffset = CGPointMake(0, textField.frame.origin.y - 30)
```

Comment:
This one pulls the scrollView up, when selecting a textfield, in order to see what you are typing.